proto/gnmi_ext/gnmi_ext.proto

```diff
@@ -159,3 +160,30 @@ message Depth {
   // returned.
   uint32 level = 1;
 }
+
+// ConfigSubscription extension allows clients to subscribe to configuration
+// schema nodes only.
+message ConfigSubscription {
+  oneof action {
+    // ConfigSubscriptionStart is sent by the server in the SubscribeRequest
+    ConfigSubscriptionStart start = 1;
+    // ConfigSubscriptionSyncDone is sent by the server in the SubscribeResponse
+    ConfigSubscriptionSyncDone sync_done = 2;
+  }
+}
+
+// ConfigSubscriptionStart is used to indicate to a target that for a given set
+// of paths in the SubscribeRequest, the client wishes to receive updates
+// for the configuration schema nodes only.
+message ConfigSubscriptionStart {}
+
+// ConfigSubscriptionSyncDone is sent by the server in the SubscribeResponse
+// after all the updates for the configuration schema nodes have been sent.
+message ConfigSubscriptionSyncDone {
+  // ID of a commit confirm operation as assigned by the client
+  // see Commit Confirm extension for more details.
+  string commit_confirm_id = 1;
```

Comment:
Should this be a `oneof` between the two values?

Reply:
The two values can both be present in the same message. The `commit_confirm_id` is present if the commit is a commit confirmed one. 
The `server_commit_id` is an internal ID to the target. Implementations can leave it empty if no ID is available or simply set it to the commit timestamp.

Reply:
fwiw, here is a demo that shows how both commit_confirm_id and server_commit_id are both present when commit with confirm is executed

https://youtu.be/MO7_ZYbRT3k?si=ukWgC4H73D8_Nqgy&t=723

Reply:
Thanks -- the specification document helped clarify when these are used.

My only comment (made elsewhere too) is whether we should have a generic "done" bool in this message for the case where neither is specified to avoid having to do a non-nil check for the oneof with an empty message. WDYT?

Reply:
I see, nothing against this if it makes things clearer. It's also inline with the `sync_update: true` message.

Reply:
great, sgtm -- it makes things clearer for me as a dozy human -- remembering semantics of having an explicit field is easier than remembering semantics of an empty message.